doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -102,91 +319,92 @@ Submit pull request
-  Chapel repository need to have all non-trivial changes reviewed. Developers
-  who have been given write access can merge trivial changes (e.g. small bug
-  fixes, documentation changes) without review.
 
-  .. note::
 
-    Ideally, someone should volunteer to review your pull request within a day
-    or two. If this doesn't happen, feel free to make some noise. Ideally the
-    review should take place within a few days, though timing may vary
-    depending on other deadlines.
+.. _Merge changes in to master:
 
-* Iterate with the reviewer until you're both satisfied. If you can't come to
-  agreement, one of you should bring other developers (individually or via
-  chapel-developers_) into the conversation to get a broader opinion (one of
-  the jobs of the reviewer is to serve as a proxy for other developers, or to
-  bring those developers into the conversation if they feel unqualified to do
-  so).
+Merge changes in to master
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * Once the pull request is approved, it can be merged. This can be done by
   either the reviewer or developer (given sufficient permissions), as decided
   between the two of them.
 
-  .. note::
+  Note: Recall that while git history can be rewritten, it is difficult and does
+  not remove any local copies that have been pulled down. So be very careful not
+  to commit anything that you might regret later (e.g., sensitive code, code
+  owned by anyone other than yourself). Ideally, the review will catch such
+  issues, but the ultimate responsibility is on the developer.
+
+* If you are reviewing code from an external contributor without push
+  privileges, please look through the `Reviewer responsibilities`_ once more
+  before merging the change.
+
+After the final version of the change has been agreed upon, navigate to the
+pull request:
+
+go to
+
+https://github.com/chapel-lang/chapel/pulls
+
+or
+
+``https://github.com/chapel-lang/chapel/pull/<number>``
+
+and click the friendly green button "Merge pull request" (it is possible to
+merge the pull request from the command line also and the pull request page has
+details). When you click "Merge pull request", you will need to enter a commit
+message. See `Final merge message`_ for a reminder on what that commit message
+should entail.
 
-    Recall that while git history can be rewritten, it is both difficult and
-    does not remove any local copies that have been pulled down. So be very
-    careful not to commit anything that you might regret later (e.g., sensitive
-    code, code owned by anyone other than yourself). Ideally, the review will
-    catch such issues, but the ultimate responsibility is on the developer.
+.. _Watch automatic testing:
+
+Watch automatic testing and address issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * In short order, a smoke-test will be run against the commit to make sure that
   nothing basic has been broken by it.  Monitor the
```

Comment:
The part about "or that they are due to a commit other than yours" could be simplified to "to ensure that your commit has not caused failures"